src/js/_enqueues/wp/updates.js

```diff
@@ -518,7 +519,7 @@
 
 		$document.trigger( 'wp-plugin-updating', args );
 
-		if ( 'plugin-information-footer' === $card.attr('id' ) ) {
+		if ( isPluginInstall && 'plugin-information-footer' === $card.attr('id' ) ) {
```

Comment:
Will this `$card` element and the `'id'` attribute _always_ exist on this `pagenow`? Is there any scenario where it might not exist?

Reply:
nit (while you're touching this code):
```suggestion
		if ( isPluginInstall && 'plugin-information-footer' === $card.attr( 'id' ) ) {
```

Reply:
>Will this $card element and the 'id' attribute always exist on this `pagenow`? Is there any scenario where it might not exist?

The Plugins Install UI should always have these cards. So the change here ensures this check is only done when on that page.

The change resolves the issue in my testing ✅

Reply:
You can see circa line 498 that `$card` is defined when it's on an install page. The definition was modified by Plugin Dependencies to add the `#plugin-information-footer` selector, but the `.'plugin-card-' + args.slug` part has been there for around 8 years.

I'm doing a little checking in case the use of `$card.attr( 'id' )` differs in some way from the `$card.removeClass()` used at line 507.

P.S. Nicely spotted on that missing space!

Reply:
Added the space in [aca780d](https://github.com/WordPress/wordpress-develop/pull/6098/commits/aca780de76bcae6e568b120dd71b757e2197886a).